emails/tests/fixtures/single_recipient_fr_expected.email

```diff
@@ -0,0 +1,146 @@
+Subject: localized email header + footer
+From: "fxastage@protonmail.com [via Relay]" <ebsbdsan7@test.com>
+To: user@example.com
+Reply-To: replies@default.com
+Resent-From: fxastage@protonmail.com
+MIME-Version: 1.0
+Content-Type: multipart/mixed; boundary="==[BOUNDARY0]=="
+
+--==[BOUNDARY0]==
+Content-Type: multipart/alternative;
+ boundary="==[BOUNDARY1]=="
+MIME-Version: 1.0
+
+--==[BOUNDARY1]==
+Content-Type: text/plain; charset="us-ascii"
+MIME-Version: 1.0
+Content-Transfer-Encoding: 7bit
+
+This email was sent to your alias ebsbdsan7@test.com. To stop receiving emails sent to this alias, update the forwarding settings in your dashboard.
```

Comment:
question (blocking): Should we test a non-ascii character in the plain text too?